docs/the-new-architecture/pillars-codegen.md

```diff
@@ -230,3 +230,59 @@ The **Codegen** for a Fabric Native Component contains a `MyFabricComponentManag
 Then, there is a layer of JNI C++ files that are used by Fabric to render the components. The basic element for a Fabric Component is the `ShadowNode`: it represents a node in the React abstract tree. The `ShadowNode` represents a React entity; therefore it could need some props, which are defined in the `Props` files and, sometimes, an `EventEmitter`, defined in the corresponding file.
 
 The **Codegen** also creates a `ComponentDescriptor.h`, which is required to get a proper handle on the Fabric Native Component.
+
+## The Codegen CLI
+
+**npx react-native codegen** [**--path** *path*] [**--platform** *string*] [**--outputPath** *path*]
+
+#### DESCRIPTION
+
+This command runs `react-native-codegen` for your project.
+The codegen must be configured as described in [Configure Codegen](../new-architecture-library-intro#configure-codegen).
+
+The following options are available:
+
+- `--path` - Path to `package.json`. The default path is current working directory.
+- `--platform` - Target platform. Supported values: `android`, `ios`, `all`. The default value is `all`.
+- `--outputPath` - Path to override `codegenCofig.outputDir`. The default value is `undefined`.
```

Comment:
if the default value is undefined... what happens? will the CLI fail? Will it crash?

By reading the doc, it feels like that this parameter should be mandatory.
If that's the case I suggest to:
1. move the outputPath at the top of the parameter list (it is also the first, alphabetically)
2. specify that the parameter is mandatory.